source/_integrations/unifi.markdown

```diff
@@ -62,7 +62,15 @@ It is recommended that you run the UniFi Network application in a dedicated virt
 
 ## Button
 
-Provides the ability to restart a UniFi device. This feature requires admin privileges.
+The Button entities will only be available and usable if the integration has an UniFi Network account with administrator privileges.
+
+### Power cycle PoE
+
+Use the **Power cycle PoE** button entity to power cycle one specific PoE port to cause the connected device to restart.
+
+### Restart UniFi device
+
+Use the **Restart UniFi device** button entity to restart the entire UniFi device. In case the device is a PoE switch PoE supply is not affected.
```

Comment:
```suggestion
Use the **Restart UniFi device** button entity to restart the entire UniFi device. In case the device is a PoE switch, the PoE supply is not affected.
```